mods/default/functions.lua

```diff
@@ -330,8 +330,9 @@ local function leafdecay_after_destruct(pos, oldnode, def)
 	for _, v in pairs(minetest.find_nodes_in_area(vector.subtract(pos, def.radius),
 			vector.add(pos, def.radius), def.leaves)) do
 		local node = minetest.get_node(v)
-		if node.param2 == 0 then
-			minetest.get_node_timer(v):start(math.random(20, 120) / 10)
+		local timer = minetest.get_node_timer(v)
+		if node.param2 == 0 and not timer:is_started() then
+			timer:start(math.random(20, 120) / 10)
```

Comment:
No, we want more randomness, not less randomness.

if you remove the /10 factor in here, then the leaves will all decay when t = 1.0, 2.0, 3.0, 4.0 etc.

Now, if people have `nodetimer_interval` set to `0.1`, the leaves will decay when t = 0.7, 1.3, 1.9, 2.6 ... etc., which is much more random.